edgehog-device-runtime-containers/src/events/mod.rs

```diff
@@ -108,66 +105,20 @@ where
             return Ok(());
         };
 
-        let mut container_id = ContainerId::new(Some(local_id), id);
-
-        let Some(inspect) = container_id.inspect(&self.client).await? else {
-            debug!("container deleted");
-
-            AvailableContainer::new(&id)
-                .send(&mut self.device, ContainerStatus::Received)
-                .await?;
-
-            return Ok(());
-        };
-
-        let Some(container_state) = inspect.state.and_then(|state| state.status) else {
-            warn!("couldn't find status in inspect container response");
-
-            return Ok(());
-        };
-
-        let status = match container_state {
-            ContainerStateStatusEnum::CREATED => ContainerStatus::Created,
-            ContainerStateStatusEnum::RUNNING | ContainerStateStatusEnum::RESTARTING => {
-                ContainerStatus::Running
-            }
-            ContainerStateStatusEnum::REMOVING => ContainerStatus::Received,
-            ContainerStateStatusEnum::EXITED | ContainerStateStatusEnum::DEAD => {
-                ContainerStatus::Stopped
-            }
-            ContainerStateStatusEnum::PAUSED | ContainerStateStatusEnum::EMPTY => {
-                debug!(%container_state, "ignoring state");
-
-                return Ok(());
-            }
-        };
-
-        AvailableContainer::new(&id)
-            .send(&mut self.device, status)
-            .await?;
+        self.refresh(Id::new(ResourceType::Container, id)).await?;
 
         Ok(())
     }
 
     #[instrument(skip(self))]
     async fn handle_image(&mut self, local_id: String) -> eyre::Result<()> {
-        let Some((id, reference)) = self.store.find_image_by_local_id(local_id.clone()).await?
-        else {
+        let Some((id, _)) = self.store.find_image_by_local_id(local_id.clone()).await? else {
             debug!("couldn't find image");
```

Comment:
```suggestion
            warn!("couldn't find image");
```

Reply:
In this file we should just emit debug since we are listening to all the container events, not just the one regarding the edgehog device runtime.